Userland/Libraries/LibWeb/DOM/Node.cpp

```diff
@@ -2325,16 +2325,13 @@ ErrorOr<String> Node::name_or_description(NameOrDescription target, Document con
             // i. Set the accumulated text to the empty string.
             total_accumulated_text.clear();
             // ii. Check for CSS generated textual content associated with the current node and include it in the accumulated text. The CSS :before and :after pseudo elements [CSS2] can provide textual content for elements that have a content model.
```

Comment:
While you're here: The current spec correctly says `::before` and `::after` with `::` instead of `:`, here and in the bullets below it. It'd be nice to change that.